components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/LoginDialogFragment.kt

```diff
@@ -128,7 +136,14 @@ internal class LoginDialogFragment : PromptDialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         view.findViewById<AppCompatTextView>(R.id.host_name).text = origin
-
+        val iconView = view.findViewById<ImageView>(R.id.host_icon)
+        if (icon != null) {
+            iconView.setImageBitmap(icon)
+        } else {
+            val tintColor = ContextCompat.getColor(requireContext(),
+                requireContext().theme.resolveAttribute(android.R.attr.textColorPrimary))
+            ImageViewCompat.setImageTintList(iconView, ColorStateList.valueOf(tintColor))
```

Comment:
Could you help me to clarify? Maybe I'm missing something :), I this section gets executed when `iconView` is null,  I'm not sure why we are trying to set the tint on it if it's already null, thanks in advance!

Reply:
It's tinting the default icon AFAICT but maybe we can set the tint for the default icon in xml?

Reply:
@Amejia481 Yep, it's for the default one. 
@ekager We definitely can do it in xml, but then I'll have to remove it for the site icons, otherwise all of them will be tinted, since it's only one ImageView for both icons/default

Reply:
Gotcha, I'm fine with this!

Reply:
@BranescuMihai in line 145 

`ImageViewCompat.setImageTintList(iconView, ColorStateList.valueOf(tintColor))`

I think we are passing a null value to `ImageViewCompat.setImageTintList` as the `else` statement will be executed when `!(icon != null)` in other words when the icon is null. Is that on purpose?

Reply:
Hmm, so the `icon`(the site bitmap) is null, but the `iconView` (the imageView) is **not** null, so `ImageViewCompat.setImageTintList` should correctly set the tint on the non-null imageView

Reply:
Sorry my bad 🤦 got confused between `iconView` and `icon`

Reply:
This one is resolved!